src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Issue Alert Configuration
+sidebar_order: 10
+redirect_from:
+ - /product/sentry-basics/guides/alert-notifications/issue-alerts/
+description: "Learn more about the options for configuring an issue alert."
+---
+
+Sentry provides several configuration options to create an issue alert based on your organization's needs.
+
+## Environment
+
+Specify which <PlatformLink to="/configuration/environments/">environment(s)</PlatformLink> will use this particular alert rule. This control filters on the `environment` tag in your events. This filter is helpful because the urgency and workflows you apply to production alerts might differ from those you apply to alerts originating from your QA environment, for example.
+
+The “Environment” dropdown list here has the same environments that are available for the selected project in the global “Environment” dropdown (this does not include hidden environments). Selecting "All Environments" is equivalent to having no environment filter.
+
+## Team
+
+You can choose a team to associate with an alert so that members of that team can edit the alert. Note that you can only make this association if you are a member of the team. If no team is selected, anyone can edit the alert.
+
+## Alert Name
+
+Give your alert a descriptive name, such as the team affected and the topic of the alert. For example, "Frontend Latency", "Backend Failure Rate", or "Billing Apdex".
+
+## "When" Conditions: Triggers
+
+"When" conditions, or triggers, specify what type of activity you'd like monitored for the issue:
+
+- First seen
+- Changes state from `resolved` to `unresolved`
+- Changes state from `ignored` to `unresolved`
+- Seen more than a certain number of times in an interval
+- Seen by more than a certain number of unique users in an interval
+
+Triggers are optional. If you don’t select a trigger, the "When" conditions are considered met by default. That is, _all_ issues will meet this condition.
```

Comment:
@adhiraj I added a sentence above to further clarify what we mean by "the 'When' conditions are considered met by default". Just confirming, it's all _issues_ that we're saying meet the criteria, correct (vs events or events and issues)?

Reply:
Yes correct, the alert is evaluated each time an _event_ is received, and it's the _issue_ for that event that meets the criteria